src/bundle/Resources/views/themes/admin/url_wildcard/list.html.twig

```diff
@@ -7,6 +7,25 @@
     }
 %}
 
+<section class="container">
+    {{ form_start(form_list, {'attr': {'class': 'form-inline justify-content-between'}}) }}
+    <div class="input-group">
+        {{ form_widget(form_list.searchQuery, { attr: {
+            'placeholder': 'url_wildcard.search.placeholder'|trans|desc('Viewing %viewing% out of %total% items'),
```

Comment:
Desc does not match the value in xliff.

Reply:
Corrected, I don't know why I was thinking about pagination